Algorithm.Python/CustomModelsAlgorithm.py

```diff
@@ -71,15 +71,19 @@ class CustomFillModel(ImmediateFillModel):
     def __init__(self, algorithm):
         self.algorithm = algorithm
         self.absoluteRemainingByOrderId = {}
-        random.seed(100)
+        self.random = Random(387510346)
 
     def MarketFill(self, asset, order):
-        #if not _absoluteRemainingByOrderId.TryGetValue(order.Id, absoluteRemaining):
         absoluteRemaining = order.AbsoluteQuantity
-        self.absoluteRemainingByOrderId[order.Id] = order.AbsoluteQuantity
+
+        if order.Id in self.absoluteRemainingByOrderId.keys():
+            absoluteRemaining = self.absoluteRemainingByOrderId[order.Id]
+            self.absoluteRemainingByOrderId[order.Id] = order.AbsoluteQuantity
```

Comment:
Is this line required `self.absoluteRemainingByOrderId[order.Id] = order.AbsoluteQuantity`? Because I think it will get removed or set in `if/else` some lines bellow

Reply:
Good catch, it seems to me it isn't required. I've gone ahead and removed it on my latest commit 👍